### PR TITLE
[Mac4ever.com] New Ruleset

### PR DESCRIPTION
**Pull request by LoHub**

### CODE REVIEW

**Comment by J0WI** [comment]
lgtm

**Comment by J0WI** [comment]
Re-open to trigger Travis

File: src/chrome/content/rules/Mac4Ever.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<ruleset name="Mac4Ever (Partial)">
+
+	<target host="mac4ever.com"/>
+	<target host="beta.mac4ever.com"/>
+	<target host="dev.mac4ever.com"/>
+	<target host="download.mac4ever.com"/>
+	<target host="forum.mac4ever.com"/>
+	<target host="iphone.mac4ever.com"/>
+	<target host="live.mac4ever.com"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Mac4Ever.xml]: Mismatch

File: src/chrome/content/rules/Mac4Ever.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	<target host="mail.mac4ever.com"/>
+	<target host="mysql.mac4ever.com"/>
+	<target host="old.mac4ever.com"/>
+	<target host="preprod.mac4ever.com"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Mac4Ever.xml]: Invalid redirect

**Comment by zoracon** [comment]
Closing due to no revisions made. Can reopen if interest for revision is indicated

**Pull request closed without merging**